src/components/faqTemplates/shared/FaqList.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import ExpandableFaq from './ExpandableFaq'
+
+const ExpandableFaqList = ({ list }) => {
+  return list
+    ? list.map(({ question, answer }) => (
```

Comment:
does this work
```suggestion
  return list?.map(({ question, answer }) => (
```

Reply:
ooo is this new in es2021?

Reply:
That makes it undefined, but I've put a `?? null` afterwards. I think it'll work.